docs/facturacion/refacturacion.md

```diff
@@ -0,0 +1,58 @@
+# Refacturació
+
+Quan hem emès una factura que no és correcte s'ha de refacturar. Bàsicament poden
+passar dues coses:
+
+  * S'ha emès una factura que **no** s'havia d'emetre.
+  * S'ha emès una factura amb errors administratius o de càlcul.
+
+!!! info
+    Per més informació sobre els tipus de factures rectificadores que hi ha
+    veure l'apartat [Tipus de factures](tipos_facturas)
```

Comment:
You may write the file path instead of the relative path for correct link build

`tipos_facturas`  →   `tipos_facturas.md`